<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>800</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_top_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_top_center">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_top_right">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_bottom_left">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bottom_center">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_bottom_right">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_font">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_top_left">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_top_center">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_top_right">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bottom_left">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bottom_center">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_bottom_right">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>520</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>63</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>84</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ext">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>155</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>157</y>
     <width>63</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>125</y>
     <width>63</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Padding</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_padding">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>95</y>
     <width>63</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_padding_font_size">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>70</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
